torch/quantization/observer.py

```diff
@@ -171,40 +197,93 @@ def _calculate_qparams(self, min_val, max_val):
 
 
 class MinMaxObserver(_ObserverBase):
-    r"""Default Observer Module
-    A default implementation of the observer module, only works for
-    `per_tensor_affine` quantization scheme.  The module will record the
-    running average of max and min value of the observed Tensor and
-    calculate_qparams will calculate scale and zero_point
+    r"""Observer module for computing the quantization parameters based on the
+    running min and max values.
+
+    This observer uses the tensor min/max statistics to compute the quantization
+    parameters. The module records the running minimum and maximum of incoming
+    tensors, and uses this statistic to compute the quantization parameters.
+
+    Args:
+        dtype: Quantized data type
+        qscheme: Quantization scheme to be used
+        reduce_range: Reduces the range of the quantized data type by 1 bit
+
+    Given running min/max as :math:`x_\text{min}` and :math:`x_\text{max}`,
+    scale :math:`s` and zero point :math:`z` are computed as:
+
+    The running minimum/maximum :math:`x_\text{min/max}` is computed as:
+
+    .. math::
+
+        \begin{array}{ll}
+        x_\text{min} &= \begin{cases}
+            \min(X) & \text{if~}x_\text{min} = \text{None} \\
+            \min\left(x_\text{min}, \min(X)\right) & \text{otherwise}
+        \end{cases}\\
+        x_\text{max} &= \begin{cases}
+            \max(X) & \text{if~}x_\text{max} = \text{None} \\
+            \max\left(x_\text{max}, \max(X)\right) & \text{otherwise}
+        \end{cases}\\
+        \end{array}
```

Comment:
![image](https://user-images.githubusercontent.com/4216323/66872677-66f67100-ef5b-11e9-9f96-efb2ca3780b9.png)

Reply:
do we need this...

Reply:
I dunno -- maybe not. I think we can just say that this is a prior min and tensor min.